### PR TITLE
Add loading state for successfull otp read

### PR DESCRIPTION
**Pull request by mehul-gohappy**

### CODE REVIEW

File: screens/loginScreen/LoginScreen.js [hunk 1]
```diff
@@ -207,19 +214,19 @@ class LoginScreen extends Component {
...
-                this._backendSignIn(
-                  user.uid,
-                  user.displayName,
-                  "https://www.pngitem.com/pimgs/m/272-2720607_this-icon-for-gender-neutral-user-circle-hd.png",
-                  user.phoneNumber
-                );
-              } catch (error) {}
-            }
-          });
+          // firebase.auth().onAuthStateChanged((user) => {
```
Comment (Rakshit1411) [inline screens/loginScreen/LoginScreen.js:217]: why ?

**Review by Rakshit1411 — CHANGES_REQUESTED** [review]

File: screens/loginScreen/LoginScreen.js (hunk 1 above)
Comment (mehul-gohappy) [reply screens/loginScreen/LoginScreen.js:217]: this onAuthStateChanged function gets triggered automatically when an OTP is recieved from firebase on the device, and if we trigger the _backend_signin manually, both of these will get triggered and one of them will produce the invalid credentials error
this function is also calling the backend_signin method inside

Comment (Rakshit1411) [reply screens/loginScreen/LoginScreen.js:217]: But this was not the intended task, right ?

Comment (Rakshit1411) [reply screens/loginScreen/LoginScreen.js:217]: This PR should only contain [Add loading state for successfull otp read]

**Review by Rakshit1411 — APPROVED** [review]

**Pull request merged**